tests/php/tests/includes/test_class.wp-job-manager-usage-tracking-data.php

```diff
@@ -44,4 +106,118 @@ public function test_employers() {
+
+		$this->assertEquals( $this->expired_count, $data['jobs_status_expired'] );
+	}
+
+	/**
+	 * Pending jobs count.
+	 *
+	 * @since 1.30.0
+	 * @covers WP_Job_Manager_Usage_Tracking_Data::get_usage_data
+	 */
+	public function test_get_usage_data_pending_jobs() {
+		$data = WP_Job_Manager_Usage_Tracking_Data::get_usage_data();
+
+		$this->assertEquals( $this->pending_count, $data['jobs_status_pending'] );
+	}
+
+	/**
+	 * Pending payment jobs count.
+	 *
+	 * @since 1.30.0
+	 * @covers WP_Job_Manager_Usage_Tracking_Data::get_usage_data
+	 */
+	public function test_get_usage_data_pending_payment_jobs() {
+		$data = WP_Job_Manager_Usage_Tracking_Data::get_usage_data();
+
+		$this->assertEquals( $this->pending_payment_count, $data['jobs_status_pending_payment'] );
+	}
+
+	/**
+	 * Preview jobs count.
+	 *
+	 * @since 1.30.0
+	 * @covers WP_Job_Manager_Usage_Tracking_Data::get_usage_data
+	 */
+	public function test_get_usage_data_preview_jobs() {
+		$data = WP_Job_Manager_Usage_Tracking_Data::get_usage_data();
+
+		$this->assertEquals( $this->preview_count, $data['jobs_status_preview'] );
+	}
+
+	/**
+	 * Published jobs count.
+	 *
+	 * @since 1.30.0
+	 * @covers WP_Job_Manager_Usage_Tracking_Data::get_usage_data
+	 */
+	public function test_get_usage_data_publish_jobs() {
+		$data = WP_Job_Manager_Usage_Tracking_Data::get_usage_data();
+
+		$this->assertEquals( $this->publish_count, $data['jobs_status_publish'] );
+	}
+
+	/**
+	 * Jobs with a company logo count.
+	 *
+	 * @since 1.30.0
+	 * @covers WP_Job_Manager_Usage_Tracking_Data::get_usage_data
+	 * @covers WP_Job_Manager_Usage_Tracking_Data::get_company_logo_count
+	 */
+	public function test_get_company_logo_count() {
+		// Create some media attachments.
+		$media = $this->factory->attachment->create_many(
+			6, array(
+				'post_type'   => 'job_listing',
+				'post_status' => 'publish',
+			)
+		);
+
+		// Add logos to some listings with varying statuses.
+		add_post_meta( $this->listings[0], '_thumbnail_id', $media[0] );
+		add_post_meta( $this->listings[5], '_thumbnail_id', $media[1] );
+		add_post_meta( $this->listings[6], '_thumbnail_id', $media[2] );
+		add_post_meta( $this->listings[12], '_thumbnail_id', $media[3] );
+		add_post_meta( $this->listings[20], '_thumbnail_id', $media[4] );
+		add_post_meta( $this->listings[24], '_thumbnail_id', $media[5] );
+
+		$data = WP_Job_Manager_Usage_Tracking_Data::get_usage_data();
+
+		// 2 expired + 1 publish
+		$this->assertEquals( 3, $data['jobs_logo'] );
+	}
+
+	/**
+	 * Jobs with a company logo count.
+	 *
+	 * @since 1.30.0
+	 * @covers WP_Job_Manager_Usage_Tracking_Data::get_usage_data
+	 * @covers WP_Job_Manager_Usage_Tracking_Data::get_job_type_count
+	 */
+	public function test_get_job_type_count() {
+		$terms = $this->factory->term->create_many( 6, array( 'taxonomy' => 'job_listing_type' ) );
+
+		// Assign job types to some jobs.
+		wp_set_object_terms( $this->listings[0], $terms[0], 'job_listing_type', false );
```

Comment:
Similar to above, it's not very clear here which jobs the types are being added to, and I think it will be brittle.